drivers/pinmux/Kconfig

```diff
@@ -30,28 +30,30 @@ config PINMUX_INIT_PRIORITY
 	  rule for particular boards. Don't change this value unless you
 	  know what you are doing.
 
-source "drivers/pinmux/Kconfig.mcux"
-
-source "drivers/pinmux/Kconfig.mcux_lpc"
-
-source "drivers/pinmux/Kconfig.stm32"
-
 source "drivers/pinmux/Kconfig.beetle"
 
-source "drivers/pinmux/Kconfig.sifive"
-
 source "drivers/pinmux/Kconfig.cc13xx_cc26xx"
 
 source "drivers/pinmux/Kconfig.esp32"
 
-source "drivers/pinmux/Kconfig.sam0"
+source "drivers/pinmux/Kconfig.hsdk"
 
 source "drivers/pinmux/Kconfig.intel_s1000"
 
+source "drivers/pinmux/Kconfig.lpc11u6x"
+
+source "drivers/pinmux/Kconfig.mcux"
+
+source "drivers/pinmux/Kconfig.mcux_lpc"
+
 source "drivers/pinmux/Kconfig.rv32m1"
 
-source "drivers/pinmux/Kconfig.xec"
+source "drivers/pinmux/Kconfig.sifive"
 
-source "drivers/pinmux/Kconfig.hsdk"
```

Comment:
Could the sorting of this file be submitted as a separate PR?

Reply:
Do you suggest to make this PR depend on another one sorting the Kconfig / CMakeLists.txt files ? Or do you rather suggest to drop this patch and eventually come back "later" with another PR to do some sorting ?

Honestly, this was the occasion to do some cleaning but we can drop this patch...

Reply:
My personal opinion is that it's OK to mix commits to do cleanup into PRs that subsequently change the file, which is what seems to be happening here.  It's not generally ok to combine cleanup with other changes in the commit , which is not happening here.

But we don't have a policy.

Reply:
OK let's keep this patch in this PR then.